Add sms functionality when user selects a section to query

ni3ol opened the pull request.

Commit:
Changed and added menuitems to give user option to receive details via sms rather than seeing over ussd first. User can also return to Main Menu, choose another section to query or exit at this point.

Commit:
Sms functionality added. Also fixed test for checking PaginatedState when location results are returned.

Commit:
Removed unused code and fixed some unneccesary spacing errors.

ni3ol commented:
@hodgestar @miltontony this code is ready for review.

Commit:
Added code from feature/issue-5-no-location-returned.

Commit:
Added return state when sms sent and fixed syntax.

Commit:
Fixed minor syntax errors.

Commit:
Changed sms state to be a delegation state to avoid sending duplicate smses.

Commit:
Changed sms message to be sent to users.

Commit:
Capatalise first letter of location name.

Commit:
Added function to capitalise location names.

Commit:
Fixed sort function for Election data. Tnx Justin.

Commit:
Fixed spacing. Ready for review.

ni3ol commented:
@hodgestar @rudigiesler @justinvdm  ready for review

Commit:
Line spacing fixed.

rudigiesler commented:
It seems like there are changes from another pull request in here too. @ni3ol would you mind merging the latest develop branch into this branch?

Commit:
Fixed merge conflicts.

ni3ol commented:
@rudigiesler have merged develop into this branch. Thanks for your help!

rudigiesler commented on test/app.test.js:
```diff
@@ -91,25 +93,95 @@ describe("app", function() {
+                it("should page 2 of location results", function() {
+                    return tester
+                        .setup.user.state('states:location')
+                        .inputs('Claremont', '3')
```
I don't think that this is quite testing what the description says it is testing. It should be `'Claremnt', '1', 'Claremont', '3'`, so that they first enter an incorrect location, and then a correct one.

rudigiesler commented on go-app.js:
```diff
@@ -22,6 +22,12 @@ go.app = function() {
+            return string.replace(/\w\S*/g, function(s){
+                return s.charAt(0).toUpperCase() + s.substr(1).toLowerCase();
+            });
+        }
```
We probably want a unit tests for this function to make sure that it's doing what we want it to do.

rudigiesler commented:
Just a few comments about tests. Also, GitHub is showing 8,302 lines deleted from the fixtures, is this intended? Looks good to me otherwise.

Commit:
Fixed test for incorrect location returned, and added test plus fixtures for testing CapitaliseLocation().

ni3ol commented:
Hi @rudigiesler, I've fixed the "incorrect location" test, and added a unit test for the capitaliseLocation() function.
8302 lines were removed because I changed the way my fixtures are displayed, so that it is easier to see all the different requests. 
Please could you review again.

rudigiesler commented:
:+1: Looks good to me.

ni3ol commented:
Thank you :)
